lib/associations/belongs-to-many.js

```diff
@@ -470,9 +470,7 @@ class BelongsToMany extends Association {
       instances = [instances];
     }
 
-    options = Object.assign({
-      raw: true
-    }, options, {
+    options = Object.assign({}, options, {
```

Comment:
Why was `raw: true` removed?

Reply:
I believe our way of using a custom data type is almost a hack.

![image](https://user-images.githubusercontent.com/4292308/62539874-e8949900-b85e-11e9-873d-fe7938df6b1d.png)

Since multiple DataTypes use a common type over DB Binary Protocol, we cannot use `.parse` and do the trick in `_sanitize`. 

Now I wonder why typecasting uses the `column type` of Binary Protocol and not the DataType  defined by Model (`column name` = `Model schema attribute`). Is it because of support for raw queries, aliases, sql functions, etc ?